Formula/pdftoedn.rb

```diff
@@ -3,6 +3,7 @@ class Pdftoedn < Formula
   homepage "https://github.com/edporras/pdftoedn"
   url "https://github.com/edporras/pdftoedn/archive/v0.34.2.tar.gz"
   sha256 "94e5888accae92380fd5e4b6a7ee4211f05814059a9f540b071a27993113be95"
+  revision 2
```

Comment:
Should be `revision 1` not 2, but are we sure the revision is even needed at all?